src/components/PlayQuizComponent.tsx

```diff
@@ -1,18 +1,37 @@
 import * as React from "react";
-import { Button, Pane, Pill } from "evergreen-ui";
-import { Quiz } from "../store/types";
+import { Button, Pane, Pill, Table } from "evergreen-ui";
+import { Quiz, Category } from "../store/types";
 import { useState, useContext } from "react";
 import { PlayersComponent } from "./PlayersComponent";
 import { ViewQuestionComponent } from "./ViewQuestionComponent";
-import { MethodsContext } from "src/store";
+import { MethodsContext } from "../store";
+
+// const calculateScore = (playerIndex: number, categories: Category[]) => {
+//   const allQuestions = categories.map((cat) => cat.questions);
+//   const flatAllQuestions = allQuestions.reduce(
+//     (acc, qList) => acc.concat(qList),
+//     []
+//   );
+//   const myQuestions = flatAllQuestions.filter(
+//     (q) => q.answererIndex === playerIndex
+//   );
+//   const myPoints = myQuestions.map((q) => q.points);
+//   return myPoints.reduce((a, b) => a + b, 0);
+// };
+
+const calculateScore = (pIndex: number, cats: Category[]) =>
+  cats
+    .flatMap((cat) => cat.questions)
+    .filter((q) => q.answererIndex === pIndex)
+    .map((q) => q.points)
+    .reduce((a, b) => a + b, 0);
 
 export const PlayQuizComponent = (props: { quiz: Quiz; qIndex: string }) => {
   const quiz = props.quiz;
-
   const [showPlayersScreen, setShowPlayersScreen] = useState<boolean>(false);
-  const { setPlayers } = useContext(MethodsContext);
+  const { setPlayers, editQuestion } = useContext(MethodsContext);
 
-  const [indexes, setIndexes] = useState<{
+  const [activeQues, setActiveQues] = useState<{
     catIndex: number;
     quesIndex: number;
   }>({ catIndex: -1, quesIndex: -1 });
```

Comment:
This can br set to `null` and instead of checking for `activeQues.catIndex` you can check for just `activeQues`